### ISSUE TITLE
Cannot repeat the result Fine-tune the unsupervised pre-trained model in task_adaptation/image_classification

### ISSUE THREAD

**Issue by ReloJeffrey** [thuml/Transfer-Learning-Library#156]
For fine-tune the unsupervised pre-trained model, Only the erm reult can be repeat, the co_tuning、bi_tuning accuray is lower than erm

**Comment by thucbx99** [comment]
Can you give a more detailed description, such as the experimental _dataset_ and the _proportion of labeled data_. And which version of pytorch are you using? It’s suggested to use **_pytorch==1.7.1_** and **_torchvision==0.8.2_** in order to reproduce the benchmark results.

**Comment by ReloJeffrey** [comment]
> Can you give a more detailed description, such as the experimental _dataset_ and the _proportion of labeled data_. And which version of pytorch are you using? It’s suggested to use **_pytorch==1.7.1_** and **_torchvision==0.8.2_** in order to reproduce the benchmark results.

I have try to run erm、co_tuning in Fine-tune the supervised pre-trained model and MoCo (Unsupervised Pretraining) for CUB-200-2011、Standford Cars、Aircrafts. I just run the .sh provided and I found all the results in the Fine-tune the supervised pre-trained model can be repeated. But for MoCo (Unsupervised Pretraining), only erm can be repeated.The results for co_tuning is lower than erm in all three dataset and all the proportion of labeled data. I have use  torch==1.7.0 with torchvision==0.8.0 and pytorch==1.7.1 with torchvision==0.8.2 both.

**Comment by thucbx99** [comment]
Thanks for providing the details. I'm running these experiments again.

**Comment by thucbx99** [comment]
I found out that this is because an important command line option `--finetune` is missing from the provided script. By specifying `--finetune` we apply a `0.1x` learning rate to the backbone and are able to reproduce the results.
